dvc/main.py

```diff
@@ -93,7 +93,7 @@ def main(argv=None):
         # so won't be reused by any other subsequent run anyway.
         clean_repos()
 
-    if ret != 0:
+    if ret != 0 and ret != 1:
```

Comment:
This will have consequences outside of `dvc check-ignore`. But on the other hand, we've heard people complain about `"Having any problems?"` message being too annoying on normal failures.

But actually we don't really need this, if `-q` works as `quiet`(logger.info should be silenced). But looking into it, I see that it is indeed broken right now, as we restore the loglevel a few lines above. This is out of scope for this PR, so let me adjust that on master so you could rebase.

Reply:
Yes, I ought to submit an issue for it. We have only one return number that not showing "Having any problems?".

Reply:
@karajan1001 Merged https://github.com/iterative/dvc/pull/4304 , please rebase on top of master, I it should be fixed.